### PR TITLE
Add ADR for file copy from source to files folder

### PR DESCRIPTION
**Pull request by balzer814**
Add ADR for file copy from source to files folder

- [x] Ensure that you followed http://eclipse.github.io/winery/dev/ToolChain#github---prepare-pull-request. Especially, we require **a single commit**
- [x] Ensure that the commit message is [a good commit message](https://github.com/erlang/otp/wiki/Writing-good-commit-messages)
- [x] Ensure to use auto format in **all** files
- [ ] Change in CHANGELOG.md described
- [ ] Tests created for changes
- [ ] Screenshots added (for UI changes)

### CODE REVIEW

**Comment by genie-winery** [comment]
Can one of the admins verify this patch?

**Comment by koppor** [comment]
Please fix ip validation - the `Sign-off:` line is needed.

File: docs/adr/0015-copy-source-to-files.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+## Decision Outcome
+* Chosen Alternative: *Just copying the sources as they are*
+* Since the decision about an external IDE in [0014-use-eclipse-orion-as-editor](0014-use-eclipse-orion-as-editor.md)
+a compiler on the winery server or a further dependency would be needed, the decision was also made under the
+consideration of the most time efficient approach which lead to the choice of alternative 2.
+
+## Pros and Cons of the Alternatives
+
+### *Compiling files before copying them*
+* `+` *With one click the user can copy and compile the files*
```
Comment (koppor) [inline docs/adr/0015-copy-source-to-files.md]: No emphasis with `*` please

File: docs/adr/0015-copy-source-to-files.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# *Copying files from the source to the files folder*
+
+## Considered Alternatives
+* *Compiling files before copying them*
+* *Just copying the sources as they are*
```
Comment (koppor) [inline docs/adr/0015-copy-source-to-files.md]: - The chosen option always goes first
- `Just` is judging, remove that word

File: docs/adr/0015-copy-source-to-files.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
+# *Copying files from the source to the files folder*
+
```
Comment (koppor) [inline docs/adr/0015-copy-source-to-files.md]: Please add a decision drivers:

- support for scripting languages
- support for compiled languages

File: docs/adr/0015-copy-source-to-files.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+## Considered Alternatives
+* *Compiling files before copying them*
```
Comment (koppor) [inline docs/adr/0015-copy-source-to-files.md]: Replace by "Require external tooling to go from source to files"

File: docs/adr/0015-copy-source-to-files.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+* *Just copying the sources as they are*
+
+## Decision Outcome
+* Chosen Alternative: *Just copying the sources as they are*
+* Since the decision about an external IDE in [0014-use-eclipse-orion-as-editor](0014-use-eclipse-orion-as-editor.md)
```
Comment (koppor) [inline docs/adr/0015-copy-source-to-files.md]: Sentence too long. The line of argumentation is: For supporting compiled languages, it is relied on an external IDE (see [ADR-0014](...)). Thus, the only left support is for scripting languages. In that case, the source can be directly used as files ("binary") in an artifact template.

To offer the same UI for editing scripting and compiled languages, the source files are always displayed in the "Source" tab.

File: docs/adr/0015-copy-source-to-files.md [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+a compiler on the winery server or a further dependency would be needed, the decision was also made under the
+consideration of the most time efficient approach which lead to the choice of alternative 2.
+
+## Pros and Cons of the Alternatives
```
Comment (koppor) [inline docs/adr/0015-copy-source-to-files.md]: Maybe that can be deleted with the new line of argumentation?

**Comment by koppor** [comment]
I fixed the style of the ADR at 5dc84b59

**Pull request merged**